assets/html/scss/documenter/components/_docstring.scss

```diff
@@ -46,7 +46,7 @@
       bottom: $documenter-docstring-body-padding-v / 2;
       @extend .tag;
       @extend .is-primary;
-      @extend .is-size-7;
+      @extend .is-size-7 !optional;
```

Comment:
Note: "The target selector was not found" is fixed here.

But this would be considered a workaround, so maybe we need to look deeper into why `.is-size-7` is not found after the update.

Reply:
It looks like we weren't including the helpers Sass at all in the light theme. I took the liberty of pushing a fix for it.